src/lib.rs

```diff
@@ -1280,7 +1280,7 @@ impl ExpressionStatus {
     }
 }
 
-mod errors {
+pub mod errors {
```

Comment:
No worries! Believe it or not you're the first duct user that I've heard from, and I'm thrilled.